guide/contribute/review.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: guide
+title: Review
+description: Contribute to the guide by reviewing a proposed change
+nav_order: 2
+parent: Contribute to guide
+permalink: /guide/contribute/review
+image: /assets/images/guide/contribute/contribute-review-preview.jpg
+---
+
+# Reviewing a proposed change
+
+Peer-review is a foundation of an open-source project. The review process happens in the open, and anyone can preview and assess a proposed change.
+
+Reviewing is a great entry point for new contributors. By inspecting others' work, you can get a better insight into the project, familiarize yourself with our processes or learn about Bitcoin.
+
+{% include youtube.html id="OXPReIz9HMs" %}
+
+## Prerequisites
+
+The review process requires only a GitHub account and no additional software. It's good practice to read our guidelines that will help you better evaluate proposed changes.
+
+1. Create a [GitHub account](https://github.com/)
+2. Read the [content guidelines](content-guidelines.md)
```

Comment:
3 links to markdown files. Should be replaced with links to pages.

Reply:
What's the difference?